Server/JokesOnYou.Web.Api/Controllers/JokesController.cs

```diff
@@ -39,6 +40,14 @@ public async Task<ActionResult<IEnumerable<JokeReplyDto>>> GetAllJokesAsync()
             return Ok(jokeDtos);
         }
 
+        [AllowAnonymous]
+        [HttpGet("Results")]
+        public async Task<ActionResult<IEnumerable<JokeReplyDto>>> GetFilteredJokesAsync([FromQuery]JokesFilterDto jokesFilter)
```

Comment:
We dont need a separate endpoint for filtering, filters in a query string are optional by default.

Reply:
cool I'll replace the GetAllJokes.